dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1632,4 +1632,6 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
 
 sendRedirect.hyper.text.url=<html><body><p>Redirecting to <a href="{0}"></a></p></body></html>
 
-query.or.post.is.null=Request query string or post data is null.
+query.or.post.is.null=CWWWC0009E: Request query string or post data is null.
```

Comment:
Change "Request query string or post data is null." to "The HTTP request query string or the HTTP POST request data is null."